OpenGD77_User_Guide.md

```diff
@@ -352,17 +372,20 @@ The CPS must be used to install the firmware and also to read and write the code
     - Choose your radio model.  
       ![firmware loader window](media/Firmware_loader-02.png)<!-- { width=420 } -->
 
-    - For Software licensing reasons the first time that you use the firmware loader you must click 'Firmware Installation Instructions' and follow the procedure described.
+      ![firmware loader window](media/Firmware_loader-03.png)<!-- { width=420 } -->
+
+    - For Software licensing reasons the first time that you use the firmware loader you must click 'Select official firmware (donor) file' and then select the donor file you previoulsy downloaded.
```

Comment:
I guess there is a typo ;-)

Reply:
I can't see any typo.  I deliberately changed the bit about firmware installation instructions because they are only relevant to the GD77 and I have included the instructions about downloading the donor file earlier in the document, so at this point the person should already have the necessary file.

Reply:
previoulsy ;-)